account_statement_import_camt/models/account_journal.py

```diff
@@ -1,13 +1,14 @@
 # Copyright 2019 ACSONE SA/NV <thomas.binsfeld@acsone.eu>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import _, models
+from odoo import models
 
 
 class AccountJournal(models.Model):
     _inherit = "account.journal"
 
     def _get_bank_statements_available_import_formats(self):
+        _ = self.env._
```

Comment:
Woah, haven't seen this one before :open_mouth:

Does this replaces `_` import in 18 version? I can see it being annoying needing to set up the variable in every method instead of just one import :thinking: